doc/notes/fomega/cek-cps-experiments/src/CekMachine.cps.hs

```diff
@@ -1,5 +1,3 @@
--- editorconfig-checker-disable-file
-
 -- This file contains a version of the CEK machine in
```

Comment:
`doc/notes/fomega` should probably be excluded wholesale, or probably just deleted tbh.

Reply:
Why would we delete `doc/notes/fomega`? It contains the entire history of us choosing Scott-encoded data types, `ifix`, mutually recursive functions and whatnot. I think it's useful to have this history, even though it's partially duplicated in the papers. And it has low to no maintenance costs.